backend-stub/setup/index.js

```diff
@@ -0,0 +1,107 @@
+const fetch = require("node-fetch");
+const uniqid = require("uniqid");
+const async = require("async");
+const { google } = require("googleapis");
+
+require("dotenv").config();
+
+module.exports = async function (context, req) {
+  const sendRequest = async (body) => {
+    const config = {
+      method: "POST",
+      mode: "cors",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(body),
+    };
+    return await fetch(process.env.SETUP_ENDPOINT, config);
+  };
+
+  const successMsg = () => {
+    context.res = {
+      body: "Your feedback module has been generated! Check your email for confirmation and further instructions.",
+    };
+  };
+
+  const errorMsg = (err) =>
+    (context.res = {
+      status: 500,
+      body: `Request error. ${err}`,
+    });
+
+  context.log("JavaScript HTTP trigger function processed a request.");
+
+  const body = req.body;
+  body.id = uniqid();
+
+  const scopes = ["https://www.googleapis.com/auth/drive"];
+  const credentials = {

```

Comment:
do we need to define all the credentials again if we are already defining them in backend-stub/index.js?

Reply:
Yes— I was only testing it in backend-stub/index.js to make sure that it works through .env— that file actually doesn't have any effect on the deployed Azure function.